### ISSUE TITLE
Installing chrome fails (exit code 2)

### ISSUE THREAD

**Issue by jnizet** [CircleCI-Public/browser-tools-orb#31]
### Orb version

1.2.1.

Works fine with 1.1 though.

### What happened
See https://app.circleci.com/pipelines/github/Ninja-Squad/globe42/2592/workflows/6f2ce57f-3969-4641-af94-fe406c1b00c0/jobs/2611

Installing Chrome fails:

Here's a minimal config to reproduce the error:

```yaml
version: 2.1
orbs:
  browser-tools: circleci/browser-tools@1.2.1
jobs: 
  build:

    working_directory: ~/globe42

    docker: 
      - image: cimg/openjdk:11.0-browsers

    steps: 
      - browser-tools/install-chrome
      - browser-tools/install-chromedriver
      - checkout
      - run: |
          java --version
          google-chrome --version
```

And here are the relevant logs:

```
Google Chrome is not currently installed; installing it
Selecting previously unselected package google-chrome-stable.
(Reading database ... 41963 files and directories currently installed.)
Preparing to unpack google-chrome.deb ...
Unpacking google-chrome-stable (92.0.4515.159-1) ...
dpkg: dependency problems prevent configuration of google-chrome-stable:
 google-chrome-stable depends on libnspr4 (>= 2:4.9-2~); however:
  Package libnspr4 is not installed.
 google-chrome-stable depends on libnss3 (>= 2:3.22); however:
  Package libnss3 is not installed.

dpkg: error processing package google-chrome-stable (--install):
 dependency problems - leaving unconfigured
Processing triggers for mime-support (3.64ubuntu1) ...
Errors were encountered while processing:
 google-chrome-stable
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Correcting dependencies... Done
The following packages will be REMOVED:
  google-chrome-stable
0 upgraded, 0 newly installed, 1 to remove and 0 not upgraded.
1 not fully installed or removed.
After this operation, 256 MB disk space will be freed.
(Reading database ... 42076 files and directories currently installed.)
Removing google-chrome-stable (92.0.4515.159-1) ...
Processing triggers for mime-support (3.64ubuntu1) ...
sed: can't read /opt/google/chrome/google-chrome: No such file or directory

Exited with code exit status 2
```

### Expected behavior

It should install chrome successfully.

**Comment by bruschke** [comment]
I am basically using the same code as shown above with the same orb version.  In my case I am splitting my tests out onto separate jobs and also use parallelism in some of those jobs so I will concurrently have about 25 separate containers running.  When the 'browser-tools/install-chrome' step executes I will randomly get the following on 1 or 2 of the containers:

```
Google Chrome is not currently installed; installing it
curl: (18) transfer closed with 72364364 bytes remaining to read
```

I sort of wish that the cimg/elixir:<< elixir-version >>**-browsers** image already had the different browsers pre-installed so that we did not have to run the install-chrome step on them, but I can understand how that would be difficult given that the browsers are constantly releasing updates.

**Comment by bhb** [comment]
I see the same issue is @bruschke - roughly every other build is failing with the message above. We're on orb version 1.2.1

**Comment by felicianotech** [comment]
investigating

**Comment by bruschke** [comment]
I pulled the code and ran it directly in my repo.  For help with investigation, my instances are entering the final else (line 129) of the install command [](https://github.com/CircleCI-Public/browser-tools-orb/blob/master/src/commands/install-chrome.yml)

**Comment by felicianotech** [comment]
I have a fix for the OP's issue in #32. Once that's merged, a new patch release of this orb will be released. That should resolve the issue.

@bruschke and @bhb, you too seem to be describing a different issue. Can one of you open an Issue for it so we can track? Information about your build image, snippets of your CircleCI config, etc can all help.

**Comment by bhb** [comment]
@felicianotech I've created a new issue: https://github.com/CircleCI-Public/browser-tools-orb/issues/33
